snowflake/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Snowflake
 
+## 4.5.2 / 2023-03-03
+
+* [Fixed] Update snowflake-connector-python to 3.0.0. See [#13926](https://github.com/DataDog/integrations-core/pull/13926).
+
 # 4.5.1 / 2023-03-02
 
 * [Fixed] Bump dependency `snowflake-connector-python` to 3.0.1. See [#14073](https://github.com/DataDog/integrations-core/pull/14073).
```

Comment:
That looks weird to bump to 3.0.0 after bumping to 3.0.1

Reply:
Thanks good catch